src/util/notifications_util.js

```diff
@@ -337,22 +338,43 @@ function get_connection(connect) {
 }
 
 
-async function test_notifications(bucket, connect_files_dir) {
-    if (!bucket.notifications) {
+async function test_notifications(notifs, nc_config_dir) {
```

Comment:
Could you add a JSDoc to this function? (to help us understand better `validate_bucket_notifications`).